src/Snippets/Docs/Chaos.Behavior.cs

```diff
@@ -0,0 +1,84 @@
+using System.Net;
+using Polly.Retry;
+using Polly.Simmy;
+using Polly.Simmy.Behavior;
+
+namespace Snippets.Docs;
+
+internal static partial class Chaos
+{
+    public static void BehaviorUsage()
+    {
+        static ValueTask RestartRedisVM() => ValueTask.CompletedTask;
+
+        #region chaos-behavior-usage
+        // Behavior using the default options.
+        // See https://www.pollydocs.org/chaos/behavior#defaults for defaults.
+        var optionsDefault = new BehaviorStrategyOptions();
```

Comment:
Just realized, the default options won't work, the `BehaviorAction` is required.

Reply:
Shall I just remove the `optionsDefault` from all sample?

Reply:
Yes, please. Except for latency, that one will work even with defaults.

Reply:
Done